template.env

```diff
@@ -78,6 +76,14 @@ NB_API_QUERY_URL=http://XX.XX.XX.XX
 # NB_QUERY_TAG=latest
 # --------------------------------------
 
+# ---- SECURITY CONFIGURATION ----
+# The below settings will be used for both the query tool and the f-API.
+# NB_ENABLE_AUTH=true
+
+# If NB_ENABLE_AUTH is set to true, you MUST provide a valid OAuth client ID for the query tool.
```

Comment:
Maybe add another line here on what that means. i.e. "please look at our docs for ..." or "to enable login with Google, please read more here ...."

Reply:
Yeah, good point. I purposefully omitted specific details in case our Google OAuth implementation is short-lived (e.g., if we switch to ORCiD soon), but I guess we can always change the template.env